core/server/api/configuration.js

```diff
@@ -30,18 +24,22 @@ function getAboutConfig() {
 
 function getBaseConfig() {
     return {
-        fileStorage:    {value: (config.fileStorage !== false), type: 'bool'},
-        useGravatar:    {value: !config.isPrivacyDisabled('useGravatar'), type: 'bool'},
-        publicAPI:      labsFlag('publicAPI'),
-        blogUrl:        {value: config.get('url').replace(/\/$/, ''), type: 'string'},
-        blogTitle:      {value: config.get('theme').title, type: 'string'},
-        routeKeywords:  {value: JSON.stringify(config.get('routeKeywords')), type: 'json'}
+        fileStorage:    config.get('fileStorage') !== false,
+        useGravatar:    !config.isPrivacyDisabled('useGravatar'),
+        publicAPI:      config.get('publicAPI') === true,
+        blogUrl:        config.get('url').replace(/\/$/, ''),
+        blogTitle:      config.get('theme').title,
+        routeKeywords:  JSON.stringify(config.get('routeKeywords'))
```

Comment:
I think we want this as raw JSON rather than stringified? If we send a stringified value with no type: 'json' then clients will treat it as a string value.